src/app/Http/Controllers/LanguageCrudController.php

```diff
@@ -65,7 +65,7 @@ public function setupCreateOperation()
         $this->crud->addField([
             'name' => 'flag',
             'label' => trans('backpack::langfilemanager.flag_image'),
-            'type' => 'browse',
+            'type' => backpack_pro() ? 'browse' : 'text',
```

Comment:
@tabacitu this was a browse field, that now belongs to backpack pro, so without backpack pro it becomes a text field.

Other than this, everything's works great 👌